BUILDING.md

```diff
@@ -1,244 +1,102 @@
 Rust 1.51.0 or higher is required.
-
 To organize build tasks, we use [cargo-make](https://sagiegurari.github.io/cargo-make/).
 To get it, run:
-
 ```shell
 cargo install cargo-make
 ```
-
+There are two paths to enable the required capabilities in docker for use with OOTB. You can modify your existing docker daemon to enable OCI support or you can install and optional container orchestration tool called Crane
 #### Docker
-
 Bottlerocket uses [Docker](https://docs.docker.com/install/#supported-platforms) to orchestrate package and image builds.
-
 We recommend Docker 20.10.10 or later.
 Builds rely on Docker's integrated BuildKit support, which has received many fixes and improvements in newer versions.
 The default seccomp policy of older versions of Docker do not support the `clone3` syscall in recent versions of Fedora or Ubuntu, on which the Bottlerocket SDK is based.
-
 You'll need to have Docker installed and running, with your user account added to the `docker` group.
 Docker's [post-installation steps for Linux](https://docs.docker.com/install/linux/linux-postinstall/) will walk you through that.
-
-> Note: If you're on a newer Linux distribution using the unified cgroup hierarchy with cgroups v2, you may need to disable it to work with current versions of runc.
-> You'll know this is the case if you see an error like `docker: Error response from daemon: OCI runtime create failed: this version of runc doesn't work on cgroups v2: unknown.`
-> Set the kernel parameter `systemd.unified_cgroup_hierarchy=0` in your boot configuration (e.g. GRUB) and reboot.
-
-### Build process
-
-To build an image, run:
-
-```shell
-cargo make
+You'll also need to enable the containerd-snapshotter and buildkit features for your docker daemon. This is required to ensure docker compatibility with OCI Images (which Kits are stored in).
+```json
+{
+  "features": {
+    "buildkit": true,
+    "containerd-snapshotter": true
+  }
+}
 ```
+#### Crane
+Crane is a container tool that the container registry team at google have published that handles oci image operations cleaner. It does not require a daemon and thus you shouldn't need the above docker features to use it. Twoliter (since 0.4.2) now supports utilizing crane instead of docker if it is installed. 
 
-This will build an image for the default variant (a recent `aws-k8s-*`, see the `BUILDSYS_VARIANT` variable in [Makefile.toml](Makefile.toml) to find the current default variant).
-All packages will be built in turn, and then compiled into an `img` file in the `build/images/` directory.
 
-The version number in [Release.toml](Release.toml) will be used in naming the file, and will be used inside the image as the release version.
-If you're planning on [publishing your build](PUBLISHING.md), you may want to change the version.
+[Installation Instructions](https://github.com/google/go-containerregistry/tree/main/cmd/crane)
```

Comment:
nit: I would work this into a sentence somewhere rather than just dropping a hyperlink here

also we should recommend `gcrane` for automatically using cloud provider credentials

Reply:
Makes sense. Worked it into a sentence instead